templates/SilverStripe/Forms/GridField/GridFieldViewButton.ss

```diff
@@ -1 +1 @@
-<a class="action action-detail view-link" href="$Link">View</a>
+<a class="grid-field__icon-action font-icon-right-open btn--icon-large action action-detail edit-link" href="$Link"><span class="sr-only">View</span></a>
```

Comment:
it's the `view-link` but has the `edit-link` class, this could be mis-interpreted later on